core/corehttp/gateway_handler.go

```diff
@@ -125,6 +125,10 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 	i.addUserHeaders(w) // ok, _now_ write user's headers.
 	w.Header().Set("X-IPFS-Path", urlPath)
 
+	//robots-header to prevent search-engines from indexing the gateway because of the duplicate-content problematic when makeing the gateway public
+	w.Header().Set("X-Robots-Tag", "noindex")
```

Comment:
i actually want the gateway to be indexed by search engines, and just opened https://github.com/ipfs/go-ipfs/issues/1751 

what is the problem with robots indexing it?